Kickstarter-iOS/Views/Controllers/ProjectPamphletViewController.swift

```diff
@@ -268,29 +271,3 @@ extension ProjectPamphletViewController: ProjectNavBarViewControllerDelegate {
     self.contentController.tableView.scrollToTop()
   }
 }
-
-// MARK: - Styles
-
-private var backThisProjectButtonStyle = { (button: UIButton) -> UIButton in
```

Comment:
Moved these styles to `CheckoutStyles.swift` so we can reuse them for all green buttons